sys/libkern/inet_ntop.c

```diff
@@ -37,7 +37,7 @@ static char	*inet_ntop6(const u_char *src, char *dst, socklen_t size);
  * author:
  *	Paul Vixie, 1996.
  */
-char *
+const char *
 inet_ntop(int af, const void *src, char *dst, socklen_t size)
```

Comment:
`inet_ntop` actually return `pointer to presentation format address (`dst'), or NULL (see errno)`. So why mark it `const char *` ?

Reply:
Posix sez it returns const char *

Reply:
> Posix sez it returns const char *

Yes. Indeed.

@rilysh  So the commit message should mention this. That will be much clear.

Reply:
Yeah I just saw that too. It seems like man pages described it right (according to POSIX).